components/Accordion/src/index.scss

```diff
@@ -34,43 +38,51 @@
   padding-block-start: var(--denhaag-accordion-panel-padding-block);
   padding-block-end: var(--denhaag-accordion-panel-padding-block);
   padding-inline-end: var(--denhaag-accordion-panel-padding-inline);
-  position: relative;
   min-height: var(--denhaag-accordion-panel-min-height);
   margin-block-start: var(--denhaag-accordion-panel-margin-block);
   margin-block-end: var(--denhaag-accordion-panel-margin-block);
-  width: var(--denhaag-accordion-panel-width);
+  font-size: var(--denhaag-accordion-panel-font-size);
 }
 
-.denhaag-accordion__panel > .denhaag-icon {
```

Comment:
Leave this be. I've added this because this was causing an issue with other icons within the panels

Reply:
@paulAcato DONE